service-rpc/src/main/java/org/apache/iotdb/rpc/IoTDBRpcDataSet.java

```diff
@@ -69,11 +64,14 @@ public class IoTDBRpcDataSet {
   public long statementId;
   public boolean ignoreTimeStamp;
 
-  public int rowsIndex = 0; // used to record the row index in current TSQueryDataSet
-
-  public TSQueryDataSet tsQueryDataSet = null;
-  public byte[] time; // used to cache the current time value
-  public byte[] currentBitmap; // used to cache the current bitmap for every column
+  public TsBlockSerde serde;
+  public List<ByteBuffer> queryResult;
+  public TsBlock curTsBlock;
+  public Long time; // used to cache the current time value
```

Comment:
```suggestion
  public long time; // used to cache the current time value
```
Actually, there is no need to save this, each time  we need to return current time, we can just get it from `curTsBlock`.